governance/board-sync.py

```
@@ -170,9 +170,13 @@ def scan_cards(repos):
             for it in batch:
                 if "pull_request" in it:  # issues 端点混入 PR——不是卡
                     continue
-                sl = [l["name"] for l in it.get("labels", []) if str(l.get("name", "")).startswith("state:")]
+                sl = sorted(l["name"] for l in it.get("labels", [])
```

Ruff (0.16.1)

[error] Ambiguous variable name: `l`

(E741)